docs/performance/m3query/index.md

```diff
@@ -0,0 +1,18 @@
+## Performance configurations
+
+Below are some common configurations related to performance for the query engine.
+
+### Query conversion LRU
+
+Before fetching data from M3DB, all queries are converted to a format that M3DB can understand. We have an LRU cache that caches the results of this
```

Comment:
Before fetching data from M3DB, all queries are converted to a format that M3DB can understand. Considering that the majority of these queries are repeated (dashboards and alerts being fairly fixed), the converted results are cached, up to a default max of 4096. To fine tune the size of the cache, the following configuration settings: